build-system/task-runner/amp-task-runner.js

```diff
@@ -110,10 +110,14 @@ async function runTask(taskName, taskSourceFilePath, taskFunc) {
  * - When a task is actually run, update root packages, load the entry point,
  *   validate usage, update task-specific packages, and run the task.
  * @param {string} taskName
- * @param {string} taskFuncName
- * @param {string} taskSourceFileName
+ * @param {string=} taskFuncName
+ * @param {string=} taskSourceFileName
  */
-function createTask(taskName, taskFuncName, taskSourceFileName) {
+function createTask(
+  taskName,
+  taskFuncName = taskName,
+  taskSourceFileName = taskName
+) {
   const tasksDir = path.join(__dirname, '..', 'tasks');
   const taskSourceFilePath = path.join(tasksDir, taskSourceFileName);
```

Comment:
Good idea to validate the inputs to `createTask()`. Instead of just a camelCase match, I've added a more generic check that will catch all types of errors. See screenshots in updated PR description.